orgues/models.py

```diff
@@ -249,6 +249,10 @@ class Orgue(models.Model):
     # Localisation
     code_dep_validator = RegexValidator(regex='^(97[12346]|0[1-9]|[1-8][0-9]|9[0-5]|2[AB])$',
                                         message="Renseigner un code de département valide")
+    latitude_validator = RegexValidator(regex='^([-]*([0-9]|[0-8][0-9])[.][0-9]*)$',
```

Comment:
```suggestion
    latitude_validator = RegexValidator(regex='^([-]?([0-9]|[0-8][0-9])[.][0-9]*)$',
```
Le [-]* va valider 0.123, -5.566, mais aussi ---7.987

Reply:
En effet. Mais le formulaire Django empêche de mettre plusieurs signes -